Additional load commands

qmfrederik opened the pull request:
- MachEncryptionInfo
- MachRunPath
- MachSourceVersion
- MachUuid

filipnavara commented on Melanzana.MachO/MachReader.cs:
```diff
@@ -329,11 +359,15 @@ private static MachObjectFile ReadSingle(FatArchHeader? fatArchHeader, MachMagic
+                    MachLoadCommandType.Rpath => ReadRunPath(loadCommandPtr, objectFile.IsLittleEndian),
                     _ => new MachCustomLoadCommand(loadCommandHeader.CommandType, loadCommandPtr.Slice(LoadCommandHeader.BinarySize, (int)loadCommandHeader.CommandSize - LoadCommandHeader.BinarySize).ToArray()),
-                });
+                }); ;
```
nit: Unnecessary `;`

filipnavara commented on Melanzana.MachO/MachWriter.cs:
```diff
@@ -301,6 +302,69 @@ private static void WriteDyldInfoCommand(Stream stream, MachLoadCommandType comm
+
+        private static void WriteRunPath(Stream stream, MachRunPath runPath, bool isLittleEndian, bool is64Bit)
+        {
+            var valueLength = 4 + Encoding.UTF8.GetByteCount(runPath.RunPath) + 1;
```
There's couple of null reference type warnings. Is the `RunPath` really supposed to be `string?`?

filipnavara reviewed (commented):
Few small comments, otherwise LGTM

qmfrederik commented:
Thanks @filipnavara . I addressed your comments and added a roundtrip test using an executable which contains a LC_RPATH command.  I couldn't easily create one which contained LC_ENCRYPTION_INFO.

Let me know if you have more feedback.

filipnavara commented:
Thanks a lot!
